pyproject.toml

```diff
@@ -105,6 +101,94 @@ test = [
     "coverage",
 ]
 
+[tool.ruff]
+target-version = "py38"  # minimum supported version
+line-length = 88  # same as Black.
+select = [
+    "A",  # flake8-builtins
+    "ANN",  # flake8-annotations
+    "ARG",  # flake8-unused-arguments
+    "B",  # flake8-bugbear
+    "BLE",  # flake8-blind-except
+    "C4",  # flake8-comprehensions
+    "D",  # pydocstyle
```

Comment:
We get a lot of errors on the test modules.  Personally I don't think we need to run this rule against any of the tests.

Reply:
Ah... the pre-commit hooks only consider the `screenpy/**` files to be `--all-files`, this is what was tripping me up.

I'll remove `tests` from the directories to ruff up.